docker/base/Dockerfile

```diff
@@ -95,6 +95,10 @@ RUN apt-get update && apt-get install -y \
     && pecl install --onlyreqdeps --force redis \
     && docker-php-ext-enable redis \
     #
+    # Install APCU
+    && pecl install apcu \
+    && docker-php-ext-enable apcu \
```

Comment:
why didn't you change the WIKIA_BASE_IMAGE_HASH value?

Reply:
fix in 87ce7506a5b